templates/Silk.NET.Templates/templates/silkgltriangle/Program.cs

```diff
@@ -0,0 +1,135 @@
+using System;
+using System.Numerics;
+using Silk.NET.Windowing;
+using Silk.NET.OpenGL;
+
+WindowOptions windowOptions = WindowOptions.Default;
+windowOptions.Title = "My Silk.NET Triangle";
+windowOptions.PreferredDepthBufferBits = 0;
+windowOptions.PreferredStencilBufferBits = 0;
+windowOptions.API = new GraphicsAPI(ContextAPI.OpenGL, new APIVersion(3, 3));
```

Comment:
Could use the `with` syntax, or at least that'd be what I'd recommend to users these days.